src/components/ButtonGroupPicker/readme.md

```diff
@@ -172,4 +172,43 @@ class ButtonGroupPickerTry extends React.Component {
 
     <ButtonGroupPickerTry />;
 
+```
+
+##### ButtonGroupPicker with shaded variant
+
+```js
+import React, { useState } from 'react';
+import { ButtonGroupPicker, ButtonOption } from 'react-rainbow-components';
+
+const ButtonGroupPickerTry = () => {
+    const [state, setState] = useState({ value:'' });
+
+    const handleOnChange = value => {
+        setState({ value });
+    }
+
+    const { value } = state;
+
+    return (
+        <div className="rainbow-p-vertical_large rainbow-align-content_center rainbow-flex_wrap">
+            <ButtonGroupPicker
+                id="button-group-picker-component-1"
```

Comment:
this id is already in use in first example, also if this example will not use the integration tests then we could not pass an unnecessary id